File: backend-dotnet/Models/State.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ClothingStore.Models
{
    [Table("states")]
    public class State
    {
        [Column("id")]
        public long Id { get; set; }
        
        [Column("name")]
        [Required]
        public string Name { get; set; } = string.Empty;
    }
}

File: genaibackend/src/main/java/com/clothingstore/entity/State.java
package com.clothingstore.entity;

import jakarta.persistence.*;

@Entity
@Table(name = "states")
public class State {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    
    @Column(nullable = false)
    private String name;
    
    // Constructors
    public State() {}
    
    public State(String name) {
        this.name = name;
    }
    
    // Getters and Setters
    public Long getId() { return id; }
    public void setId(Long id) { this.id = id; }
    
    public String getName() { return name; }
    public void setName(String name) { this.name = name; }
}